### ISSUE TITLE
Workbench should verify that the content type specified in the config exists

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench#375]
When the content type configured in `content_type` doesn't exist, Workbench errors out with "Error: Workbench cannot retrieve field definitions from Drupal. Please confirm that the Field, Field Storage, and Entity Form Display REST resources are enabled." This error message masks the fact that the specified content type doesn't exist.

Workbench should ping the specified content type (if there's a way to do that) and clearly indicate that it doesn't exist.

**Comment by ajstanley** [comment]
I was poking around at this and couldn't find a way to check to see if a content type exists or not.  We could write a custom endpoint, but changing the error message to 
```
        message = (
            '\nWorkbench cannot retrieve field definitions from Drupal.\n'
            'Please confirm that the Field, Field Storage, and Entity Form Display REST resources are enabled\n'
            f"and that the content type '{config['content_type']}' has been defined on {config['host']}."
        )
```
ought to give the user all the information they need.

**Comment by ajstanley** [comment]
another idea!  (I've tested this and it works)  at the start of get_entity_fields we stick the following check -
```
    type_check = issue_request(config, 'GET', f"{config['host']}/admin/structure/types/manage/{config['content_type']}")
    if type_check.status_code == 404:
        message = f"Content type {config['content_type']} not defined on {config['host']}."
        logging.error(message)
        sys.exit('Error: ' + message)
```
I can stick this in a PR if you like

**Comment by mjordan** [comment]
Yes please. I'm not sure if that check would ever be used anywhere else, but maybe it's worth putting it into a `ping_content_type` function like https://github.com/mjordan/islandora_workbench/blob/main/workbench_utils.py#L569 and  https://github.com/mjordan/islandora_workbench/blob/main/workbench_utils.py#L545 ?

**Comment by mjordan** [comment]
Requests to `admin/structure/types/manage/`, which is called from the `ping_content_type()` function, are leaving "Access denied" warnings in the Drupal log, even for the admin user. The complete error message is:

`Path: /admin/structure/types/manage/islandora_object. Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException: The used authentication method is not allowed on this route. in Drupal\Core\EventSubscriber\AuthenticationSubscriber->onExceptionAccessDenied() (line 135 of /var/www/html/drupal/web/core/lib/Drupal/Core/EventSubscriber/AuthenticationSubscriber.php).`

So we might need to find another way to verify that a content type exists.

**Comment by mjordan** [comment]
Resolved with c4ae438d110bf933292cc1152e87f15f21606aa4.
